app/services/batch_invite_job.rb

```diff
@@ -37,4 +38,9 @@ def result_of_inviting user
       'Invited!'
     end
   end
+
+  def complitely_delete_deleted_users
+    User.deleted.delete User.deleted.where(email: @invites.flat_map{|org_id, email| email.downcase}).pluck(:id)
```

Comment:
Why not use the `really_delete` method that [the docs talk about](https://github.com/rubysherpas/paranoia)?